COLLABORATOR_GUIDE.md

```diff
@@ -0,0 +1,92 @@
+# Collaborator Guide
+
+## Table of Contents
+
+* [Releases](#releases)
+  * [Getting dependencies](#getting-dependencies)
+    * [npm access](#npm-access)
+    * [GitHub personal access token](#github-personal-access-token)
+  * [Deciding what type of version release](#deciding-what-type-of-version-release)
+  * [Doing a release](#doing-a-release)
+* [Doc credit](#doc-credit)
+
+## Releases
+
+Releasing http-streaming is partially automated through various scripts.
+To do a release, you need a couple of things: npm access, GitHub personal access token.
+
+Releases are done on npm and GitHub and eventually posted on the CDN.
+These are the instructions for the npm/GitHub releases.
+
+### Getting dependencies
+
+#### npm access
+
+To see who currently has access run this:
+
+```sh
+npm owner ls @videojs/http-streaming
+```
+
+If you are a core committer, you can request access to npm from one of the current owners.
+Access is managed via an [npm organization][npm org] for [Video.js][vjs npm].
+
+#### GitHub personal access token
+
+This is used to make a GitHub release on videojs. You can get a token from the [personal access tokens](https://github.com/settings/tokens) page.
+
+After generating one, make sure to keep it safe because GitHub will not show the token for you again. A good place to save it is Lastpass Secure Notes.
+
+### Deciding what type of version release
+
+Since we follow the [conventional changelog conventions][conventions], all commits are prepended with a type, most commonly `feat` and `fix`.
```

Comment:
forgot the link for `conventions`?

Reply:
Here's a handy link: https://github.com/videojs/conventional-changelog-videojs/blob/master/convention.md